### DISCUSSION TITLE
Does the package support silent installation?

### DISCUSSION THREAD

**Discussion by Dragon1573** [MCJack123/craftos2#333] [Q&A] [answered]
Hi @MCJack123 , I'm working on contributing manifests of Craft-OS 2 to [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs). Related issues and pull requests are as follows:

- https://github.com/microsoft/winget-pkgs/issues/115055
- https://github.com/microsoft/winget-pkgs/pull/123714
- https://github.com/microsoft/winget-pkgs/pull/123726

When testing in Windows Sandbox, `winget.exe` successfully installed the package unattendedly. But the installation wizard automatically launch the main executable and ask for an "Allow analytics?" approval.

![image](https://github.com/MCJack123/craftos2/assets/49941141/497f01df-0151-4937-91e7-2d7fe944b007)

`wingetcreate.exe` inspect that the installation wizard is created by InnoSetup. Is there any options or switches available to prevent this "auto launch"? Or is there any options or switches available to silently allow / deny analytics?

**Comment by MCJack123** [comment, marked as answer]
Unfortunately it does not appear to be possible to stop the program from automatically launching. There is an option in Inno Setup's source code to disable auto-launching; however, this interferes with the auto-updater, which expects the app to relaunch after updating. I also cannot disable the analytics box for user privacy reasons. I may investigate alternate solutions involving runtime variables if possible, but for the time being you will have to tell users to close the app manually.
